docs/README.md

```diff
@@ -1,11 +1,27 @@
+| [Working with EventSources and EventCounters](EventSourceAndCounters.md) | Guidance on adding event tracing to a library                           | Contributors needing to add event tracing for diagnostics purposes                                                          |
+| [Tests on Helix](Helix.md)                                               | An overview of the Helix test environment                               | Contributors debugging tests in Helix or looking to understand the output from Helix builds                                 |
+| [Issue management](IssueManagementPolicies.md)                           | Overview of policies in place to manage issues                          | Community members and collaborators looking to understand how we handle closed issue, issues that need author feedback, etc |
+| [Package archives](PackageArchives.md)                                   |                                                                         |                                                                                                                             |
+| [Preparing a patch update](PreparingPatchUpdates.md)                     | Documentation on how to setup for a patch release of ASP.NET Core       | Collaborators looking to publish servicing updates                                                                          |
+| [Project properties](ProjectProperties.md)                               | Overview of configurable MSBuild properties on the repo                 | Contributors looking to modify how a project is packaged                                                                    |
+| [How references are resolved](ReferenceResolution.md)                    | Overview of dependency reference setup in the repo                      | Contributors looking to understand how package references are configured in the repo                                        |
+| [Servicing changes](Servicing.md)                                        | Documentation on how to submit servicing PRs to previous releases       | Collaborators to submit patches or backports to prior releases, contains the "Shiproom Template"                            |
+| [Shared framework](SharedFramework.md)                                   | Overview of the ASP.NET Core Shared framework                           | Contributors looking to understand the policies in place for managing the code of the shared framework                      |
+| Submodules                                                               |                                                                         |                                                                                                                             |
```

Comment:
Left this empty because I don't think we need docs on submodules here. We don't use them aggressively and this time of stuff is better suited for the official Git docs.

Reply:
Curious why a non link instead of just no mention?

Reply:
My plan was to either document it or delete the file and the link altogether if folks felt it wasn't necessary. So kept it in here for now to start the discussion. I'm in favor of deleting the file...

Reply:
Seems fine to leave if we already have a doc written for it, it doesn't hurt does it?

Reply:
I know I've definitely randomly seen submodule issues at times when switching branches and what not, so its useful still in those scenarios for people

Reply:
Please leave this documentation and fill out this line in the table